_topic_maps/_topic_map.yml

```diff
@@ -456,9 +456,11 @@ Dir: support
 Distros: openshift-enterprise,openshift-online,openshift-origin
 Topics:
 - Name: Support overview
-  File: index  
+  File: index
 - Name: Getting support
   File: getting-support
+- Name: Managing your cluster resources
+  File: managing-cluster-resources
```

Comment:
Can you help me understand the reason for all the changes besides these two lines to the topic map? I suspect that the rest of the changes are part of a rebase operation but I can't tell. For this PR, I would only expect to see changes here for adding the "Managing your cluster resources" section.

Reply:
Everything else appears to be okay but I really can't tell because there are two many additions and deletions in the `_topic_map.yaml` file for me to know for sure.

Reply:
You were right. I hadn't rebased properly. Now _topic_map.yaml is showing content alright.